packages/site/src/components/FeedbackBanner.tsx

```diff
@@ -19,63 +25,60 @@ export const FeedbackBanner = () => {
 
   return (
     <div
+      data-elevation="elevated"
       style={{
         position: "fixed",
         bottom: 0,
         width: "375px",
         backgroundColor: "var(--color-surface)",
-        color: "white",
         left: 0,
         right: 0,
         margin: "0 auto",
         zIndex: 2,
         display: "flex",
         flexDirection: "column",
         border: "1px solid var(--color-border)",
+        borderBottom: "none",
         borderTopLeftRadius: "var(--radius-base)",
         borderTopRightRadius: "var(--radius-base)",
         boxShadow: "var(--shadow-base)",
         padding: "var(--space-small)",
-        transition: "transform var(--timing-base) ease-in-out",
-        transform: isExpanded
-          ? "translateY(0px)"
-          : "translateY(calc(100% - 40px))",
```

Comment:
Disclosure just does this now (we lose a bit of smoothness but tbh I think it's fine)